src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonSerializerSettingsProvider.cs

```diff
@@ -20,6 +21,11 @@ public static JsonSerializerSettings CreateSerializerSettings()
         {
             return new JsonSerializerSettings
             {
+                ContractResolver = new DefaultContractResolver
```

Comment:
Are any of the tests serializing objects with dictionaries (e.g. IDictionary<string, string>) or overriden property names (e.g. JsonProperty("name")).

If not then you should add some.